docs/data/data-grid/events/events.md

```diff
@@ -2,36 +2,72 @@
 title: Data Grid - Events
 ---
 
-# Data grid - Events [<span class="plan-pro"></span>](https://mui.com/store/items/mui-x-pro/)
+# Data grid - Events
 
-<p class="description">The data grid emits events that can be subscribed to attach custom behavior.</p>
+<p class="description">Subscribe to the events emitted by the data grid to trigger custom behavior.</p>
 
 ## Subscribing to events
 
-You can subscribe to one of the [events emitted](/x/react-data-grid/events/#catalog-of-events) by calling `apiRef.current.subscribeEvent()` with the name of the event and a handler. The handler will be called with three arguments:
+You can subscribe to one of the [events emitted](/x/react-data-grid/events/#catalog-of-events) by providing an event handler to the grid.
 
-1. an object with information related to the event
-2. a `MuiEvent` containing the DOM event or the React synthetic event, when available
-3. a `GridCallbackDetails` containing the `GridApi` only if `DataGridPro` is being used.
+The handler is a method that will be called with three arguments:
+
+1. the parameters containing the information related to the event.
+2. the `MuiEvent` containing the DOM event or the React synthetic event, when available.
+3. the `GridCallbackDetails` containing the `GridApi` only if `DataGridPro` or `DataGridPremium` is being used.
```

Comment:
No need for periods since this is a list of incomplete sentences.

```suggestion
1. the parameters containing the information related to the event
2. the `MuiEvent` containing the DOM event or the React synthetic event, when available
3. the `GridCallbackDetails` containing the `GridApi` only if `DataGridPro` or `DataGridPremium` is being used
```